### ISSUE TITLE
Uncaught error when selecting too many attachment files

### ISSUE THREAD

**Issue by fernandoabolafio** [decred/politeiagui#1254]
- Go to the proposal submission form
- Click on "Attach a file"
- Select many files (more than 5 at least) 

After selecting them an uncaught error will show up in the console log:

```
Uncaught TypeError: Cannot read property 'maximagesize' of undefined
    at helpers.js:94
    at Array.filter (<anonymous>)
    at d (helpers.js:93)
    at helpers.js:79
    at Array.reduce (<anonymous>)
    at helpers.js:79
    at o (helpers.js:23)
    at Object.handleFilesChange [as handleFiles] (FilesField.js:32)
    at FileReader.i.onloadend (index.js:129)
```

**Comment by camus-code** [comment]
Good catch @fernandoabolafio. Just pushed a PR to address this.

As a side note, I'm thinking of opening an issue for overall image error handling. Errors are not being cleared from the state whenever the user removes the image(s) in question. 

Perhaps this issue falls in line with #1238. @victorgcramos what do you think? Does this fit the scope of what you're working on, or should I file another issue?

**Comment by victorgcramos** [comment]
Hey @camus-code! I have just opened a PR #1265 that fixes #1238! The problem that caused the issue is a little bit different. But thanks for the support!

Cheers!
